docs/13_0_0/components/colorpicker.md

```diff
@@ -22,21 +33,80 @@ ColorPicker is an input component with a color palette.
 | id | null | String | Unique identifier of the component
 | rendered | true | Boolean | Boolean value to specify the rendering of the component, when set to false component will not be rendered.
 | binding | null | Object | An el expression that maps to a server side UIComponent instance in a backing bean
-| value | null | Object | Value of the component.
+| accesskey | null | String | Access key that when pressed transfers focus to the input element.
+| alt | null | String | Alternate textual description of the input field.
+| autocomplete | null | String | Controls browser autocomplete behavior.
 | converter | null | Converter/String | An el expression or a literal text that defines a converter for the component. When it’s an EL expression, it’s resolved to a converter instance. In case it’s a static text, it must refer to a converter id
-| immediate | false | Boolean | When set true, process validations logic is executed at apply request values phase for this component.
-| required | false | Boolean | Marks component as required.
-| disabled | false | Boolean | Marks component as disabled.
-| validator | null | MethodExpr | A method expression that refers to a method for validation the input.
-| valueChangeListener | null | ValueChangeListener | A method binding expression that refers to a method for handling a valuchangeevent.
-| requiredMessage | null | String | Message to be displayed when required field validation fails.
 | converterMessage | null | String | Message to be displayed when conversion fails.
+| dir | null | String | Direction indication for text that does not inherit directionality. Valid values are LTR and RTL.
+| disabled | false | Boolean | Disables input field
+| immediate | false | Boolean | When set true, process validations logic is executed at apply request values phase for this component.
+| inputmode | null | String | Hint at the type of data this control has for touch devices to display appropriate virtual keyboard.
+| label | null | String | A localized user presentable name.
+| lang | null | String | Code describing the language used in the generated markup for this component.
```

Comment:
Not a `locale` attribute as we have on other components?

Reply:
I left locale off because in Coloris it's a global setting not an instance setting. So if you set it on one colorPicker then all on that page you will be in that language.